src/Uno.Extensions.Http/SimpleAuthenticationToken.cs

```diff
@@ -9,5 +9,5 @@ public class SimpleAuthenticationToken : IAuthenticationTokenProvider
 	public string? AccessToken { get; set; }
 
 	/// <inheritdoc />
-	public Task<string> GetAccessToken() => Task.FromResult(AccessToken ?? string.Empty);
+	public Task<string> GetAccessToken(HttpRequestMessage? requestMessage = default, CancellationToken cancellationToken = default) => Task.FromResult(AccessToken ?? string.Empty);
```

Comment:
The `HttpRequestMessage` I feel doesn't need to be added. This feels like a specific signature from the Refit side of things.